src/esssans/beam_center_finder.py

```diff
@@ -94,33 +102,38 @@ def iofq_in_quadrants(
         The quadrants are named 'south-west', 'south-east', 'north-east', and
         'north-west'.
     """
-    data = sample.copy(deep=False)
-    norm = norm.copy(deep=False)
-
-    # Offset the position according to the input shift
-    center = _offsets_to_vector(data=data, xy=xy, graph=graph)
-    data.coords['position'] = data.coords['position'] - center
-    norm.coords['position'] = data.coords['position']
-
     pi = sc.constants.pi.value
     phi = data.transform_coords(
         'phi', graph=graph, keep_intermediate=False, keep_inputs=False
     ).coords['phi']
     phi_bins = sc.linspace('phi', -pi, pi, 5, unit='rad')
     quadrants = ['south-west', 'south-east', 'north-east', 'north-west']
 
-    providers = [to_Q, merge_spectra, normalize]
+    providers = [
+        to_Q,
+        merge_spectra,
+        normalize,
+        iofq_denominator,
+        mask_wavelength,
+        detector_to_wavelength,
+        calibrate_positions,
+        solid_angle_rectangular_approximation,
+    ]
     params = {}
+    params[UncertaintyBroadcastMode] = UncertaintyBroadcastMode.upper_bound
     params[WavelengthBands] = wavelength_range
     params[QBins] = q_bins
     params[ElasticCoordTransformGraph] = graph
+    params[BeamCenter] = _offsets_to_vector(data=data, xy=xy, graph=graph)
```

Comment:
This shows an interesting "shortcoming" of Sciline: We want some specific subset of providers, basically something that is obtained by replacing sub-branches with concrete parameters. Is this indicating that we need a different interface, or helper?

Basically we want to set a parameter, and it should drop all providers that are directly or indirectly used to compute this?

Or maybe what we have here is fine, since it is explicit?

Reply:
Not sure I really understand the issue here. Isn't what you have fine?

Reply:
It is indeed fine, just slightly verbose, and easy to forget updating if we even change the main workflow.